### ISSUE TITLE
fan.js shall be updated to use the binarySwitch model

### ISSUE THREAD

**Issue by gvancuts** [01org/SmartHome-Demo#82]
The OCF server implemented in `fan.js` shall use the [`binarySwitch`](http://oneiota.org/revisions/1393) data model.

**Comment by gvancuts** [comment]
I'm wondering whether using [`AirFlow`](http://oneiota.org/revisions/1795) might be a better choice than `binarySwitch` here... thoughts?

**Comment by ttzeng** [comment]
In the OIC Smart Home device spec, the fan device oic.d.fan should contain a binary switch resource. I prefer binary switch than AirFlow.

**Comment by gvancuts** [comment]
I agree, let's follow the Smart Home specs, I did not know this was in there. Thanks!
